modules/ROOT/pages/_partials/sync/js-sync-js-basic-concepts.adoc

```diff
@@ -1,8 +1,12 @@
-= Basic concepts
+= Introduction
 
-Client is basing on Apollo GraphQL client that can be used with various web and mobile frameworks.
-We provide version for web and Apache Cordova.
-For basic concepts about Apollo GraphQL please refer to documentation for your own platform.
+The {sync-service} Javascript client SDK is built on top of the popular link:https://www.apollographql.com/docs/react[Apollo GraphQL Javascript client], and it adds the following additional features: 
```

Comment:
@wei-lee  have we agreed this name? "{sync-service} Javascript client SDK"

Reply:
maybe not, do you have a name you prefer?

Reply:
{data-client-sdk} or {voyager-client} ;) 
ie, lets assume that it could change at some point. I think the community definitely need to be involved in the naming and that may be changed for downstream

Reply:
how about `{sync-client-sdk}`? Do we have it defined somewhere?